python/python/bystro/ancestry/train.py

```diff
@@ -454,27 +662,28 @@ def _get_mi_df(train_X: pd.DataFrame, train_y_pop: pd.Series) -> pd.DataFrame:
     return mi_df
 
 
-def _serialize_data(variants: list[str], pca: PCA, rfc: RandomForestClassifier) -> None:
+def serialize_model_products(variants: Sequence[str], pca: PCA, rfc: RandomForestClassifier) -> None:
+    """Serialize variant list, pca and rfc to disk as .txt, .skops files."""
     variants_fpath = ANCESTRY_MODEL_PRODUCTS_DIR / "variants.txt"
     pca_fpath = ANCESTRY_MODEL_PRODUCTS_DIR / "pca.skop"
     rfc_fpath = ANCESTRY_MODEL_PRODUCTS_DIR / "rfc.skop"
 
     with variants_fpath.open("w") as f:
         f.write("\n".join(variants))
-    skops.io.dump(pca, pca_fpath)
-    skops.io.dump(rfc, rfc_fpath)
+    skops_dump(pca, pca_fpath)
+    skops_dump(rfc, rfc_fpath)
 
 
 def main() -> None:
     """Train global ancestry model."""
     err_msg = "This module isn't production-ready yet!"
     raise NotImplementedError(err_msg)
     genotypes, labels = _load_dataset()
-    train_X, test_X, train_y, test_y = _make_train_test_split(
+    train_X, test_X, train_y, test_y = make_train_test_split(
```

Comment:
Do you need to repeat making the test/train split here since there's a version on line 465? Or is it set up differently because it's a different model?

Reply:
yes, train.py is a bit of a junk drawer at the moment-- a lot of it was experimental code that I amnestied just to get it on github.  After deploying the chip model, glad to come back and clean up what we no longer need.

NB: we're publicizing this method because we'll need to import it into a separate script for training the chip model (to come shortly)